File: tools/findCircleArea.py
def findCircleArea (a):
    if a < 0:
        return -1
    elif a >= 0:
        pi = 3.14159
        asqrd = a ** 2
        total = 0
        total = asqrd * pi
        return round(total,3)

print(findCircleArea(-12))
print(findCircleArea(3))

File: tools/findCircleArea.js
function findCircleArea (a) {
    if (a < 0) {
        return -1;
    } else {
        var pi = 3.14159
        var asqrd = a ** 2
        var total = 0
        total = asqrd * pi
        return total.toFixed(5)
    }
}
console.log(findCircleArea(-12))
console.log(findCircleArea(3))

